gammapy/cube/core.py

```diff
@@ -324,6 +320,22 @@ def solid_angle_image(self):
         sky_map = SkyMap.from_image_hdu(image_hdu)
         return sky_map.solid_angle()
 
+    def sky_image(self, slicepos):
+        """Slice 3-dim cube into a 2-dim `~gammapy.image.SkyMap along slicepos`.
```

Comment:
Actually, @OlgaVorokh - Could you please split out the `SkyCube.sky_image` method into a separate PR and do this before continuing with this one?

In the same PR you can also remove the standalone `cube_to_image` function
https://github.com/gammapy/gammapy/blob/01b3d066ea2035ff8d8d81569c371087e2daf3a6/gammapy/cube/utils.py#L107
and update the one place where it's called: `SkyCube.solid_angle`.